vnc/vnc.c

```diff
@@ -1422,6 +1447,11 @@ lib_mod_set_param(struct vnc *v, const char *name, char *value)
     {
         v->delay_ms = g_atoi(value);
     }
+    else if (g_strcasecmp(name, "guid") == 0)
```

Comment:
lib_mod_set_param is not getting its params from config file only,
please look @ https://github.com/jsorg71/xrdp/blob/ed862ea361e1a364f153da2bd22c402f529b547e/xrdp/xrdp_mm.c#L568

This is the api for modules also for internal use.